app-frontend/src/app/components/map/annotateToolbar/annotateToolbar.module.js

```diff
@@ -16,11 +16,10 @@ const AnnotateToolbarComponent = {
 
 class AnnotateToolbarController {
     constructor(
-        $log, $scope, $ngRedux,
+        $scope, $ngRedux,
```

Comment:
Minor, but there's no need to have the args split across multiple lines unless we end up going over the 100 char limit for that line.